kvm/src/lib.rs

```diff
@@ -26,11 +28,12 @@ use libc::{open, EINVAL, ENOSPC, O_RDWR};
 use kvm_gen::*;
 use memory_model::MemoryMapping;
 use sys_util::{errno_result, Error, EventFd, Result};
-use sys_util::{
-    ioctl, ioctl_with_mut_ptr, ioctl_with_mut_ref, ioctl_with_ptr, ioctl_with_ref, ioctl_with_val,
-};
+use sys_util::{ioctl, ioctl_with_ref, ioctl_with_val};
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Shouldn't this ioctl wrappers be platform independent?

Reply:
They are only used in functions that are `x86_64` specific. That means that building this module on other platforms than `x86_64` would result in a warning.

Reply:
I removed all labelling as it was not related to this PR. Take a look.